src/services/dexes/sovryn/SovrynPriceFetcher.ts

```diff
@@ -33,13 +33,13 @@ export type PricesResponse = {
   timestamp: BigNumber;
 };
 
-export type PairRequest = {
+export type InputParams = {
```

Comment:
```suggestion
export type SovrynPriceFetcherInputParams = {
```

if we exporting, let's use better naming

Reply:
apply better naming to all places like this